standard/ERCs/erc-6900.md

```diff
@@ -484,6 +484,7 @@ During validation uninstallation, the account MUST correctly clear flags and oth
 
 - the account MUST clear all flags for the validation function, like `isGlobal`, `isSignatureValidation`, and `isUserOpValidation`.
 - the account MUST remomve all hooks and SHOULD clear hook module states by calling `onUninstall` with the user-provided data for each hook, including both pre validation hooks and execution hooks, if required by user.
+  - the account MAY ignore the revert from `onUninstall` with try/catch depending on the design principle of the account.
```

Comment:
Thoughts on this @erc6900/working-group ?

Reply:
Seems fine. Adds some flexibility to the account.

Reply:
👍 It provides developers with flexibility when a clean state is not required in certain modules